### PR TITLE
Change shortcuts

### PR DESCRIPTION
**Pull request by oskarrough**
- Changed shortcut for going to homepage from <kbd>gh</kbd> to <kbd>gg</kbd>
- Removed shortcut <kbd>gf</kbd> to go to /feedback
- Changed shortcut for going to favorites from <kbd>gs</kbd> to <kbd>gf</kbd>
- New: both <kbd>gf</kbd> (favorites) and <kbd>gt</kbd> (tracks) are now contextual. That is, they will prioritize using the channel you are browsing, then your own channel.

### CODE REVIEW

**Comment by hugurp** [comment]
Still cannot fix it, but on Firefox latest (linux) the `/` shortcut does not work as it opens the `quick find`.

File: app/mixins/keyboard-shortcuts-global.js [hunk 1]
```diff
@@ -48,8 +48,11 @@ export default Mixin.create(EKMixin, {
 	toggleShortcutsModal: on(keyUp('shift+Slash'), function() {
 		this.toggleProperty('uiStates.showShortcutsModal')
 	}),
-	focusSearchInput: on(keyUp('Slash'), function() {
-		document.querySelector('.InputAutocomplete .aa-input').focus()
+
+	focusSearchInput: on(keyDown('Slash'), function(event) {
```
Comment (hugurp) [inline app/mixins/keyboard-shortcuts-global.js]: omg! ty

Comment (hugurp) [reply app/mixins/keyboard-shortcuts-global.js]: why though keyDown, sometimes I am remotely incapable of understanding why

Comment (oskarrough) [reply app/mixins/keyboard-shortcuts-global.js]: it's the web 🤷‍♂️

Comment (hugurp) [reply app/mixins/keyboard-shortcuts-global.js]: 50014544_306923893291641_7822178740816838656_n

**Pull request merged**